R/general_meanplot.R

```diff
@@ -40,39 +42,65 @@ general_meanplot <- function(data,
       DOSNO %in% selected_cycles,
       if ("EVID" %in% names(data)) EVID == 0 else TRUE,
       NRRLT > 0
-    ) %>%
-    # rename(id_variable = id_variable) %>%
+    )
+
+  timeunit <- unique(preprocessed_data$RRLTU)
+  concunit <- unique(preprocessed_data$AVALU)
```

Comment:
issue: please use snake_case for variable names
```suggestion
  time_unit <- unique(preprocessed_data$RRLTU)
  conc_unit <- unique(preprocessed_data$AVALU)
```